addons/account/tests/test_tour.py

```diff
@@ -34,3 +34,6 @@ def test_01_account_tour(self):
                 .write({'l10n_latam_use_documents': False})
 
         self.start_tour("/web", 'account_tour', login="admin")
+
+    def test_01_account_tax_groups_tour(self):
+        self.start_tour("/web", 'account_tax_group', login="admin")
```

Comment:
@juliusc2066 
If we only install the "account" module, this tour will consistently fail because [this xmlid](https://github.com/odoo/odoo/blob/0f90e80f9e33c850026d64c9788d40563e2ec9cf/addons/account/static/tests/tours/tax_group_tests.js#L17) will not be found, unless the "account_accountant" module is already installed.

Reply:
Should be addressed in https://github.com/odoo/odoo/pull/125937